kfp/pipeline_generator/pipeline.ptmpl

```diff
@@ -0,0 +1,123 @@
+# NOTE: This file is auto generated by Pipeline Generator.
+
+import kfp.compiler as compiler
+import kfp.components as comp
+import kfp.dsl as dsl
+from kfp_support.workflow_support.utils import (
+    ONE_HOUR_SEC,
+    ONE_WEEK_SEC,
+    ComponentUtils,
+)
+__compute_import__
+
+# the name of the job script
+EXEC_SCRIPT_NAME: str = "__script_name__"
+PREFIX: str = "__prefix_name__"
+
+# components
+base_kfp_image = "__kfp_base_image__"
+
+# path to kfp component specifications files
+component_spec_path = "__component_spec_path__"
+
+# compute execution parameters. Here different tranforms might need different implementations. As
+# a result, instead of creating a component we are creating it in place here.
+compute_exec_params_op = comp.func_to_container_op(
+    func=__compute_func_name__, base_image=base_kfp_image
+)
+# create Ray cluster
+create_ray_op = comp.load_component_from_file(component_spec_path + "createRayClusterComponent.yaml")
+# execute job
+execute_ray_jobs_op = comp.load_component_from_file(component_spec_path + "__execute_comp__")
+# clean up Ray
+cleanup_ray_op = comp.load_component_from_file(component_spec_path + "deleteRayClusterComponent.yaml")
+
+# Task name is part of the pipeline name, the ray cluster name and the job name in DMF.
+TASK_NAME: str = "__pipeline_name__"
+
+
+# Pipeline to invoke execution on remote resource
+@dsl.pipeline(
+    name=TASK_NAME + "-ray-pipeline",
+    description="__pipeline_description__",
+)
+def __pipeline_name__(
+    ray_name: str = "__pipeline_name__-kfp-ray",  # name of Ray cluster
+    ray_head_options: str = '{"cpu": 1, "memory": 4, "image": "__transform_image__",\
+             "image_pull_secret": "__image_pull_secret__",  "image_pull_policy": "Always"}',
+    ray_worker_options: str = '{"replicas": 2, "max_replicas": 2, "min_replicas": 2, "cpu": 2, "memory": 4, "image_pull_secret": "__image_pull_secret__",\
```

Comment:
Can you please put the setting of the task image in the top of the file as in current pipelines (i.e., `task_image = "quay.io/dataprep1/data-prep-kit/noop:0.8.0"` and reference it in this section? The `reconcile-requirements` Makefile target expects this syntax for updating the task tag. Thanks